apps/aechannel/src/aesc_fsm.erl

```diff
@@ -2024,7 +2089,7 @@ check_op_error_msg(Op, #{ channel_id := ChanId
     case aesc_offchain_state:get_fallback_state(State) of
         {LastRound, State1} -> %% same round
             Log1 = log_msg(rcv, Op, Msg, Log),
-            {ok, ErrCode, D#data{state = State1, log = Log1}};
+            {ok, ErrCode, D#data{state = State1, log = Log1, op = ?NO_OP}};
```

Comment:
I see this changes the semantics of `check_op_error_msg/3` a bit but this actually fixes a bug :)